contents/handbook/content-and-docs/docs.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Docs ownerhship
+sidebar: Handbook
+showTitle: true
+---
+
+- **Product teams** are responsible for ensuring their docs are up-to-date. This means:
+  - Documenting new features when they're launched.
```

Comment:
I think we can remove the periods at the end of all these nested bullet point lines?